docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -13,34 +13,39 @@ Constructor
 ~~~~~~~~~~~
```

Comment:
I merged your PR about adding gif file. So, you can add the link of it.

Reply:
I added the gif link and removed the png.